build.sh

```diff
@@ -3,8 +3,10 @@ set -eu
 
 MYDIR=$(cd "$(dirname "$(readlink -f -n "${BASH_SOURCE[0]}" )" )" && pwd -P)
 
-export COMPILER=${COMPILER:?"Please set COMPILER environment variable [gnu|intel]"}
-export CMAKE_Platform=linux.${COMPILER}
+if [ -z ${CMAKE_Platform+x} ]; then
```

Comment:
@DusanJovic-NOAA please have a look at this proposed change and let me know if it is ok for you. Using this if test allows for the following on platforms supported by the ufs-weather-model:
```
module use $PWD/modulefiles/hera.intel/fv3
./build.sh
```
One could of course also require the user to set `CMAKE_Platform` directly and remove the `COMPILER` logic.

Reply:
Looks correct to me. We should always try to make build systems as simple as possible. I also agree that ``COMPILER`` can be removed, since it looks like it is only used to set  ``CMAKE_Platform``. I have no idea why I set it up this way. I guess I just reused the script I was using for gnu make builds.